README.md

```diff
@@ -6,11 +6,38 @@
 
 ## Motivation
 
-Before the Devin Status Checker extension, users had to manually monitor their Devin AI sessions, often switching back and forth between tasks to check if Devin was waiting for input. This was little bit inefficient. 
+Before the Devin Status Checker extension, users had to manually monitor their Devin AI sessions, often switching back and forth between tasks to check if Devin was waiting for input. This was little bit inefficient.
 
 To address this, the Devin Status Checker Chrome extension was developed. It automates the monitoring process by checking the status of Devin sessions in the background. When Devin's status changes to 'Devin is awaiting', the extension sends an immediate notification. This allows users to focus on other tasks with the assurance that they won't miss the moment Devin requires their attention. By streamlining this process, the extension enhances productivity and integrates AI assistance into the user's multitasking routine seamlessly.
 
 ## Features
 
 - Automatic checking of Devin status at regular intervals.
 - Notifications to alert the user when Devin is awaiting their input.
+
+## How to Install Extension
+
+To install the Devin Status Checker extension, follow these steps:
+
+1. Download the extension package from the [Releases](https://github.com/sotayamashita/devin-status-checker/releases) page.
```

Comment:
To Devin. I want a user to clone and then build extension instead of using Releases because there is no Releases.